lambdas/indexer/app.py

```diff
@@ -91,19 +92,49 @@ def post_notification():
             notify_queue = queue(config.notify_queue_name)
             notify_queue.send_message(MessageBody=json.dumps(message))
             log.info("Queued notification %r", notification)
-        return {"status": "done"}
+        return chalice.app.Response(body='', status_code=http.HTTPStatus.ACCEPTED)
     else:
         test_mode_error = f'Ignored notification {notification}. This indexer is currently in TEST MODE.'
         log.error(test_mode_error)
         raise chalice.ChaliceViewError(test_mode_error)
 
 
+def validate_request_syntax(notification):
+    try:
+        match_ = notification['match']
```

Comment:
Why is there an underscore at the end of `match_`?

```suggestion
        match_ = notification['match']
```

Reply:
PyCharm suggestion. 😅